HW_3/task_3.py

```diff
@@ -0,0 +1,33 @@
+"""
+Задача 7.
+Пользователь вводит строку текста.
+? Подсчитайте сколько раз встречается
+каждая буква в строке без использования
+метода count и с ним.
+? Результат сохраните в словаре, где ключ —
+символ, а значение — частота встречи
+символа в строке.
+? Обратите внимание на порядок ключей.
+Объясните почему они совпадают
+или не совпадают в ваших решениях.
+"""
+
+
+def frequency_dict():
+    data = "".join(item for item in (input("Enter some text: ")) if item.isalpha())
+    dictionary = {}
+
+    # using .count()
+    for elem in data:
+        if elem not in dictionary:
+            dictionary[elem] = data.count(elem)
```

Comment:
Так что вам не использовать условий можно сделать так.
```suggestion
    for elem in set(data):
        dictionary[elem] = data.count(elem)
```
Либо используйте метод `get`, как мы делали на семинарах.